### ISSUE TITLE
Investigate use of default credentials within RS environment

### ISSUE THREAD

**Issue by scott-aquia** [CDCgov/prime-reportstream#16351]
# DevSecOps Issue

## Summary

The Java keystore appears to be using the default password.

## Environment

- [ ] Local
- [X] Dev
- [X] Stage
- [X] Prod

## Priority

- [ ] **Critical** - affecting prod systems
- [ ] **Major** - blocking major functionality, deployment, etc
- [X] **Minor** - improvements, bug fixes
- [ ] **Nice-to-have** - feature request

### Blocks the following

- This issue is not blocking anything in itself.

### Blocked by the following

- Changing the keystore password will require changes to be communicated to all customers. 

## Contact

**Team**: DevSecOps

**Slack Team Channel**:

**Slack Contact (Full Name):** Scott Zimmerman

## Definition of Done

- [ ]

## Notes

- NIST 800-53 v5.1 section IA-5 - Authenticator Management states the following:

"Control: Manage system authenticators by:
...
e. Changing default authenticators prior to first use;
...
Authenticators include passwords, cryptographic devices, biometrics, certificates, one- time password devices, and ID badges. Device authenticators include certificates and passwords."
